crates/bevy_reflect_compile_fail_tests/tests/reflect_derive/custom_where_fail.rs

```diff
@@ -13,10 +13,10 @@ impl<T> FromType<T> for ReflectMyTrait {
 // Reason: populated `where` clause must be last with #[reflect(MyTrait)]
 #[derive(Reflect)]
 #[reflect(where T: std::fmt::Debug, MyTrait)]
+//~^ ERROR: expected
```

Comment:
```suggestion
//~^ ERROR: /expected.+:/
```

I'm not sure if that's the correct way to specify Regex, but the pattern should work in the case. [View it on regex101.](https://regex101.com/r/a4lEyx/1) It basically matches `expected` exactly, then 1 or more of any character, then the `:` colon.